modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -395,11 +401,16 @@ export default {
       if (this.focused) {
         return;
       }
-      this.state.add.top.show = true;
-      this.state.add.bottom.show = true;
+      if (!this.foreign) {
+        this.state.add.top.show = true;
```

Comment:
The + buttons are never desirable in a foreign widget (but are desirable in its closest non-foreign ancestor).